Formula/pipgrip.rb

```diff
@@ -47,10 +49,17 @@ class Pipgrip < Formula
   end
 
   def install
-    virtualenv_install_with_resources
+    venv = virtualenv_create(libexec, Formula["python@3.8"].opt_bin/"python3")
+    venv.pip_install resources
+    venv.pip_install buildpath
+
+    gcc_path = Formula["gcc"].bin
```

Comment:
```suggestion
    gcc_path = Formula["gcc"].opt_bin
```

So it won't break on gcc updates.